kem/xwing/xwing.go

```diff
@@ -252,7 +252,9 @@ func (pk *PublicKey) EncapsulateTo(ct, ss, seed []byte) {
 	copy(ekx[:], seed[32:])
 
 	x25519.KeyGen(&ctx, &ekx)
-	x25519.Shared(&ssx, &ekx, &pk.x)
+	if !x25519.Shared(&ssx, &ekx, &pk.x) {
+		// TODO Propagate invalid public key error, https://github.com/dconnolly/draft-connolly-cfrg-xwing-kem/issues/28
```

Comment:
`panic(err)` in the meantime.

Reply:
Since it could be triggered remotely, I didn't want to add a `panic`. Garbage in, garbage out seemed acceptable. @bwesterb WDYT?